docker/Dockerfile

```diff
@@ -1,22 +1,25 @@
-FROM debian:jessie
+FROM frolvlad/alpine-glibc
```

Comment:
Is the glibc alpine required over the standard one?  If so, perhaps a comment explaining so.